src/routes/(app)/product/[id]/+page.server.ts

```diff
@@ -177,7 +177,9 @@ async function addToCart({ params, request, locals }: RequestEvent) {
 
 	await addToCartInDb(product, quantity, {
 		user: userIdentifier(locals),
-		...(product.payWhatYouWant && { customPrice }),
+		...((product.payWhatYouWant || (product.hasVariations && product.standalone)) && {
+			customPrice
+		}),
```

Comment:
When variations are enabled, the user can decide to pay 0 € for the product

Instead, the price should be calculated from `chosenVariations`